src/helix-ui/elements/HXMenuElement.js

```diff
@@ -1,193 +1,29 @@
 import { HXElement } from './HXElement';
-import { getPosition } from '../utils/position';
-import debounce from 'lodash/debounce';
 
-const DEFAULT_POSITION = 'bottom-start';
+import { mix } from '../utils';
+import { Positionable } from '../mixins/Positionable';
 
-/**
- * Fires when the element is concealed.
- *
- * @event Menu:close
- * @since 0.6.0
- * @type {CustomEvent}
- */
-
-/**
- * Fires when the element is revealed.
- *
- * @event Menu:open
- * @since 0.6.0
- * @type {CustomEvent}
- */
+class _ProtoClass extends mix(HXElement, Positionable) {}
 
 /**
  * Defines behavior for the `<hx-menu>` element.
  *
- * @emits Menu:close
- * @emits Menu:open
  * @extends HXElement
+ * @extends Positionable
  * @hideconstructor
  * @since 0.2.0
  */
-export class HXMenuElement extends HXElement {
+export class HXMenuElement extends _ProtoClass {
     static get is () {
         return 'hx-menu';
     }
 
+    /** @override */
     $onCreate () {
-        this._onDocumentClick = this._onDocumentClick.bind(this);
-        this._onDocumentScroll = this._onDocumentScroll.bind(this);
-        this._reposition = this._reposition.bind(this);
-
-        this._onWindowResize = debounce(this._reposition, 50);
-    }
+        super.$onCreate();
+        this.DEFAULT_POSITION = 'bottom-start';
 
-    $onConnect () {
-        this.$upgradeProperty('open');
-        this.$upgradeProperty('position');
-        this.$upgradeProperty('relativeTo');
-
-        this.$defaultAttribute('position', DEFAULT_POSITION);
         this.$defaultAttribute('role', 'menu');
```

Comment:
Based on what I've had to fix in #408, attribute manipulation should reside within `$onConnect()`. Please update.

Reply:
ok will do 👍